apps/web/src/core/components/CodeBlock.tsx

```diff
@@ -1,16 +1,12 @@
 import { IconCheck, IconClipboard, IconDownload } from "@tabler/icons-react"
-import { FC, memo, useState } from "react"
+import { type FC, memo, useState } from "react"
```

Comment:
Any way we could make this code responsive to dark mode as well? A bit ironic if only the code is in light-mode :)

Reply:
Done!